### ISSUE TITLE
Images with aria-hidden are getting flagged for missing alt text.

### ISSUE THREAD

**Issue by brownhead** [Khan/tota11y#51]
It's my understanding that an `img` tag with `aria-hidden="true"` set doesn't need alt text (since it'll be invisible to screen readers anyways). The "Image alt-text" plugin flags such images anyways though.

**Comment by jdan** [comment]
Yep you're right. We cycle through `$("img")`, but should be filtering out anything hidden like ADT does.

In reality we should probably just call `audit("imagesWithoutAltText")` from ADT instead of writing our own logic. We did this to get some extra customizability (showing different annotations for presentational images), but we can probably do that after the fact.

**Comment by jdan** [comment]
FWIW `role="presentation"` or `alt=""` might be your solution here, but it shouldn't be failing regardless.
